tsinfer/inference.py

```diff
@@ -35,6 +35,7 @@
 import threading
 import time
 import time as time_
```

Comment:
This is old, but I'm not sure why we're importing ``time`` twice?

Reply:
Weird, I think it's safe to remove that!